api/envoy/config/filter/network/http_connection_manager/v2/http_connection_manager.proto

```diff
@@ -299,10 +299,8 @@ message HttpConnectionManager {
   // The maximum request headers size for incoming connections.
   // If unconfigured, the default max request headers allowed is 60 KiB.
   // Requests that exceed this limit will receive a 431 response.
-  // The max configurable limit is 96 KiB, based on current implementation
-  // constraints.
   google.protobuf.UInt32Value max_request_headers_kb = 29
```

Comment:
mind adding a release note (docs/root/version_history/current.rst) about the change?

Reply:
Added release notes